src/test/java/com/ge/predix/metering/filter/NuregoIT.java

```diff
@@ -56,85 +89,237 @@
 
     @Value("${NUREGO_INSTANCE_ID}")
     private String nuregoInstanceId;
+    
+    @Value("${NUREGO_OAUTH_URL}")
+    private String nuregoOauthURL;
+    
```

Comment:
The Nurego Oauth url can be constructed from the nuregoURL, we don't need to use two different environment variables.